src/db_state.rs

```diff
@@ -22,6 +23,15 @@ impl SSTableHandle {
         }
         false
     }
+
+    pub(crate) fn estimate_size(&self) -> u64 {
+        let info = self.info.borrow();
+        // this is a hacky estimate of the sst size since we don't have it stored anywhere
+        // right now. Just use the filter's offset and add the filter length. Since the filter
```

Comment:
Seems you're using index offset and length, not the filter. That is correct, but I think the comment needs to be updated.

Reply:
yeah, this is from before the patch to move the index out of the footer. Will update.